src/utils/findGitmojiCommand.ts

```diff
@@ -0,0 +1,86 @@
+// @flow
+import { CommitModes } from '@constants/commit'; // Assuming COMMIT_MODES is exported as CommitModes
+import FLAGS from '@constants/flags';
+
+interface Options {
```

Comment:
If we're not extending interfaces I would suggest changing this to `type`

```suggestion
type Options {
```

Reply:
I guess it all comes down to personal preference as inherently there's no wrong or correct here. But generally there's a clear distinction between using `interface` vs `type`. Structuring objects? Use `interface`. Defining tuples or unions? Use `type`.

Reply:
Also this might as well just be `type Options = Record<string, any>`

Reply:
> But generally there's a clear distinction between using interface vs type. Structuring objects? Use interface. Defining tuples or unions? Use type.

Not sure why or who defines that clear distinction, but from my point of view I would much rather go for `type` than `interface` unless I plan to extend it, favouring consistency and falling back to `type` by default unless inheritance is needed

Reply:
> Not sure why or who defines that clear distinction
I probably should have worded that differently. It's just how I've seen it used in most code bases and personally use it like that. Again, it all comes down to personal preference and I give you the freedom to decide how you want to set the coding style.